docs/TSG.md

```diff
@@ -17,22 +17,19 @@ This document is meant to be a step-by-step guide for trouble shooting any issue
 1. [I am getting an error code I don't understand.](#understanding-error-codes)
 2. [The connection is unexpectedly shutting down.](#why-is-the-connection-shutting-down)
 3. [No application (stream) data seems to be flowing.](#why-isnt-application-data-flowing)
+4. [Why is this API failing?](#why-is-this-api-failing)
 
 ## Understanding Error Codes
 
 Some error codes are MsQuic specific (`QUIC_STATUS_*`), and some are simply a passthrough from the platform. You can find the MsQuic specific error codes in the platform specific header ([msquic_posix.h](../src/inc/msquic_posix.h), [msquic_winkernel.h](../src/inc/msquic_winkernel.h), or [msquic_winuser.h](../src/inc/msquic_winuser.h)).
 
 From [msquic_winuser.h](../src/inc/msquic_winuser.h):
 ```C
-#ifndef ERROR_QUIC_HANDSHAKE_FAILURE
-#define ERROR_QUIC_HANDSHAKE_FAILURE    _HRESULT_TYPEDEF_(0x80410000L)
-#endif
-
-#ifndef ERROR_QUIC_VER_NEG_FAILURE
-#define ERROR_QUIC_VER_NEG_FAILURE      _HRESULT_TYPEDEF_(0x80410001L)
-#endif
-
-...
+#define QUIC_STATUS_ADDRESS_IN_USE          HRESULT_FROM_WIN32(WSAEADDRINUSE)               // 0x80072740
```

Comment:
nit: do you want to add a link to [QUIC_STATUS.md#well-known-status-codes](https://github.com/microsoft/msquic/blob/fa4e0d4375dc4573ce718f829ac1713622bdcda8/docs/api/QUIC_STATUS.md#well-known-status-codes) so it is easy to see the error description of each message

Reply:
Thanks for the suggestion! Will do.